WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormTableViewDataSource.swift

```diff
@@ -299,3 +315,27 @@ private extension ProductFormTableViewDataSource {
                                 showsSeparator: false))
     }
 }
+
+
+// MARK: Configure rows in Options CTA Section
+//
+private extension ProductFormTableViewDataSource {
+    func configureCellInOptionsCTASection(_ cell: UITableViewCell, row: ProductFormSection.OptionsCTARow) {
+        switch row {
+        case .addOptions:
+            configureAddOptions(cell: cell)
+        }
+    }
+
+    func configureAddOptions(cell: UITableViewCell) {
+        guard let cell = cell as? BasicTableViewCell else {
+            fatalError("Unexpected cell type \(cell) for view model: \(viewModel)")
```

Comment:
I'd suggest we replace the view model in this error message with details about the cell itself:

```suggestion
            fatalError("Unexpected cell type \(cell) for Add Options row")
```